packages/legend-extension-dsl-data-space-studio/style/index.scss

```diff
@@ -14,3 +14,31 @@
  * limitations under the License.
  */
 @forward 'data-space-template-query-promotor';
+
+@use 'mixins' as *;
+
+:root {
+  --color-text-element: var(--color-blue-50);
+}
+
+.dataSpace-editor--dark {
```

Comment:
lets add this to a separate css file and import here to index.scss

Reply:
moved